File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/GaitAnalysisFormViewModel.ts
//$1E5828E9
import { BaseViewModel } from "NebulaClient/Model/BaseViewModel";
import { GaitAnalysisForm } from "NebulaClient/Model/AtlasClientModel";
import { Type } from 'NebulaClient/ClassTransformer';

export class GaitAnalysisFormViewModel extends BaseViewModel {
    protected __type__: string = 'Core.Models.GaitAnalysisFormViewModel, Core';
    @Type(() => GaitAnalysisForm)
    public _GaitAnalysisForm: GaitAnalysisForm = new GaitAnalysisForm();
}

File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/GaitAnalysisFormViewModel.cs
//$3700963E
using System;
using System.Linq;
using Core.Models;
using Infrastructure.Filters;
using Infrastructure.Models;
using TTInstanceManagement;
using TTObjectClasses;
using TTDefinitionManagement;
using TTUtils;
using Core.Controllers;

namespace Core.Controllers
{
    public partial class GaitAnalysisFormServiceController
    {
    }
}

namespace Core.Models
{
    public partial class GaitAnalysisFormViewModel: BaseAdditionalInfoFormViewModel
    {
        public override BaseAdditionalInfo AddViewModelToContext(TTObjectContext objectContext)
        {
            var gaitAnalysisFormService = new GaitAnalysisFormServiceController();
            gaitAnalysisFormService.GaitAnalysisFormInternal(this, objectContext);
            return (GaitAnalysisForm)objectContext.GetObject(this._GaitAnalysisForm.ObjectID, "GaitAnalysisForm");
        }
    }
}
